eradiate/scenes/measure/_radiancemeter.py

```diff
@@ -67,6 +70,10 @@ def _target_origin_validator(self, attribute, value):
     def film_resolution(self) -> t.Tuple[int, int]:
         return (1, 1)
 
+    def __attrs_post_init__(self):
```

Comment:
```suggestion
    def __attrs_pre_init__(self):
```
No need to bother constructing the object if it's not allowed.